src/recovery/ffi.rs

```diff
@@ -29,7 +29,8 @@ impl RecoverableSignature {
     /// Create a new (zeroed) signature usable for the FFI interface
     pub fn new() -> RecoverableSignature { RecoverableSignature([0; 65]) }
     /// Create a new (uninitialized) signature usable for the FFI interface
-    pub unsafe fn blank() -> RecoverableSignature { mem::uninitialized() }
+    #[deprecated(since = "0.15.2", note = "Please use the new function instead")]
```

Comment:
this one still says 0.15.2

Reply:
damn forgot that there's 2 `ffi.rs` files :/